### PR TITLE
Update support.html to address errors and shutdown

### PR DESCRIPTION
**Pull request by superg2009**
updates to dates and suggestion to join discord as TRU is closed

### CODE REVIEW

File: support.html [hunk 1]
```diff
@@ -9,15 +9,15 @@ <h1>Support Lab</h1>
         lower level classes with their assignments and topics. All the hours, and the current status, of the support
         lab can be found on this page.</p>
 
-    <h3>Winter 2019</h3>
-    <p>The support lab is now <strong>open</strong> for the Winter 2019 semester. Drop by OM1365 during one of the
-        scheduled sessions, or email us at <a href="mailto:supportlab@trucsclub.ca">supportlab@trucsclub.ca</a>.</p>
+    <h3>Winter 2020</h3>
+    
+    <p>The support lab is currently <strong> Closed on campus </strong> for the Winter 2020 semester due to the TRU campus shutdown. Find us on <a href="//trucsclub.ca/discord">Discord</a> though</p>
```
Comment (frogcrush) [inline support.html]: What about "However, you can still get support online by joining our Discord chatroom! Find out how at https://trucsclub.ca/discord"

or something.

File: support.html [hunk 2]
```diff
@@ -9,15 +9,15 @@ <h1>Support Lab</h1>
...
     <iframe
         src="https://calendar.google.com/calendar/embed?title=Support%20Lab%20Schedule&amp;showCalendars=0&amp;showTz=0&amp;mode=WEEK&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=77ab0ir4p41on5bu65onbgmd3o%40group.calendar.google.com&amp;color=%230F4B38&amp;ctz=America%2FVancouver"
         style="border-width:0;width:100%" height="500" frameborder="0" scrolling="no"></iframe>
 
     <h3>Know your stuff? Volunteer with the Support Lab!</h3>
-    <p>The Winter 2019 call for volunteers is open! Each semester, students depend on the support lab for help with
+    <p>The Fall 2020 call for volunteers is open! Each semester, students depend on the support lab for help with
```
Comment (frogcrush) [inline support.html]: Are we sure? We don't even know if we'll be back at school

Comment (superg2009) [reply support.html]: I assumed , that and well its better than last winter being there.

Comment (frogcrush) [reply support.html]: Hmm. Perhaps we could say that the call for volunteers is currently closed, however you can join our Discord to help others and join our mailing list to be notified when it re-opens?

Comment (frogcrush) [reply support.html]: We should probably get some input from others.

Comment (superg2009) [reply support.html]: not a bad idea pm saying it is closed, thought about that but wasn't sure how to edit it. Advice would be good though

Comment (frogcrush) [reply support.html]: "Currently, due to school closure the call for volunteers is now closed. However, if you're still interested in helping other students, please come to our Discord chatroom (hyperlink those to page) where we are continuing an online-only version of the help lab. Additionally, you can sign up for our mailing list to get notified when our in-person help lab re-opens."

File: support.html (hunk 1 above)
Comment (superg2009) [reply support.html]: applied suggestion

File: support.html (hunk 2 above)
Comment (superg2009) [reply support.html]: good idea, put that in new commit

**Comment by superg2009** [comment]
too old at this point, will need to rewrite anyway

**Pull request closed without merging**